content/Mobile_Dev/WebAuthn/IOS/Register_Only_Security_Keys.adoc

```diff
@@ -0,0 +1,120 @@
+= Register Only Security Key on IOS Safari
+
+== I only want my user to register using a security key
+In this section we are going to walk through implementing an experience where a user is required to use a security key to register a WebAuthn credential to their account. While it is best practice to be highly permissive in the types of authenticators you allow in your application, there may be instances where you want to only permit the use of a cross-platform authenticator, especially in the space of enterprise applications. 
+
+You may want to take this approach if you don’t want your user to leverage the copyable passkeys that are created when a user uses Face ID to create their account.
+
+This is also a good flow to prompt users to register a credential that can be used across multiple devices. This aids greatly with account recovery on unfamiliar or untrusted devices during a user's initial login. 
```

Comment:
Consider using a bulleted list here to help the reader see under what conditions this UX is best for

Reply:
Added in latest commit